### DISCUSSION TITLE
Support for different raspberry pi models

### DISCUSSION THREAD

**Discussion by drashna** [rkolbi/My_Mars2Pro_Accessories#4] [Object / Parts on this hub]
Would you be willing to add support for other  raspberry pi models for the case covers?

**Comment by rkolbi** [comment]
It shouldn't be too difficult, what version rPi?

**Reply by drashna** [reply]
The ~4B~ 3A+ would probably be best.

**Reply by rkolbi** [reply]
I know 4b can do OTG, but not sure on 3A+. The easiest really is the rPiW, wiring/power is much simpler.
Curious, why do you not want to use the rPiW?

**Reply by drashna** [reply]
I meant to get back to you sooner! 
The 3A+ can't do OTG properly, but it does work.  I actually have a 3A+ hooked up, atm. 

As for the rPi0W, basically it doesn't like my access point, and doesn't like to stay connected.

**Reply by drashna** [reply]
Just a heads up, the Raspberry Pi Zero 2 Wireless was recently released, and use the same size and shape of the zero.  It works perfectly in the existing case, and works great with mariner.

**Reply by rkolbi** [reply]
Great to hear! Hard to actually get hold of one at the moment...

**Reply by ddivins** [reply]
I'd love to see the STEP files for the parts.  I have a Pi Zero 2 and would like to expand the cover a bit and add better openings for a Camera cable and maybe heat insets for the screws-- not really sure how its intended to work today.  Anyway, thanks for all the effort on this.

**Reply by jpreculeau** [reply]
hello.
Is the Raspberry PI Zéro 2W compatible ? Because the webpart is not working. (Samba is and the pi is seen as a usb key without problem)
